### DISCUSSION TITLE
Ordinal Variables in Mixed Models

### DISCUSSION THREAD

**Discussion by delucalab** [easystats/parameters#593] [General]
My apologies for having two separate discussions going at once (this question was a bit too unrelated to include in the other thread). If one of my variables (ie lesion type) is derived by looking at qualitative levels of inflammation (low/found nowhere, medium/only found in the rim, and high/found everywhere) that exist in a sort of fluid continuum would this mean that the lesion type variable is ordinal rather than categorical? Would that change the best statistical approach from a mixed model to something else?

**Comment by bwiernik** [comment]
See here for a good tutorial on fitting models with ordinal predictors and response variables https://journals.sagepub.com/doi/abs/10.1177/2515245918823199
